csrankings-x.csv

```diff
@@ -29,6 +29,7 @@ Xiabing Zhou,Soochow University,http://web.suda.edu.cn/zhouxiabing,NOSCHOLARPAGE
 Xiahai Zhuang,Fudan University,http://www.sdspeople.fudan.edu.cn/zhuangxiahai,XlavIMcAAAAJ
 Xiali Hei,University of Louisiana - Lafayette,http://www.xialihei.com,PzflF_cAAAAJ
 Xian Du,University of Massachusetts Amherst,https://mie.umass.edu/faculty/xian-du,8iO-e8MAAAAJ
+Xian Fang,Florida State University,https://www.cs.fsu.edu/~fan/index.html,Cy91GecAAAAJ

```

Comment:
As far as I can tell, this is _not_ the correct DBLP entry for this person.

Reply:
I am sorry, I mis-spelled her name. 

Please see her CV here https://www.cs.fsu.edu/~fan/Resume_08212020.pdf 
It lists the same publication.